schema/url/computes/betaDiv.raml

```diff
@@ -1,23 +1,23 @@
 #%RAML 1.0 Library
 
-uses:
-  req: ../computes.raml
-
 types:
+
   BetaDivPluginRequest:
-    type: req.ComputeRequestBase
+    type: ComputeRequestBase
     properties:
-      config: BetaDivPluginConfig
+      config: BetaDivComputeConfig
 
-  BetaDivPluginConfig:
-    type: object
-    additionalProperties: false
+  BetaDivComputeConfig:
+    type: ComputeConfigBase
     properties:
-      outputEntityId:
-        type: string
       collectionVariable: VariableSpec
       betaDivDistanceMethod: BetaDivDistanceMethod
 
   BetaDivDistanceMethod:
     type: string
-    enum: ['bray', 'jaccard', 'jsd']
\ No newline at end of file
+    # RD: what happened to these (were in data service)? 'unifrac', 'wunifrac', 'jensen-shannon'
```

Comment:
I don't think we're ready for those. I have an old memory, which may be incorrect, that they rely on building a phylogenetic tree. That's added complexity and brings up questions we decided weren't critical we had answers to for a first release of beta div.